### ISSUE TITLE
Improve test logging by removing a noisy message

### ISSUE THREAD

**Issue by brunoocasali** [meilisearch/meilisearch-dart#140]
The message `Using Meilisearch server on http://localhost:7700 for running tests.` makes it harder to understand the test running logs.

```bash
➜  meilisearch-dart git:(feature/tenant-token) dart test --concurrency=1                            
00:00 +0: test/get_keys_test.dart: Keys When has master key responds with all keys                                                  
Using Meilisearch server on http://localhost:7700 for running tests.
00:00 +1: test/get_keys_test.dart: Keys When has master key gets a key from server by key/uid                                       
Using Meilisearch server on http://localhost:7700 for running tests.
00:00 +2: test/get_keys_test.dart: Keys When has master key creates a new key                                                       
Using Meilisearch server on http://localhost:7700 for running tests.
00:00 +3: test/get_keys_test.dart: Keys When has master key creates a new key with expiresAt                                        
Using Meilisearch server on http://localhost:7700 for running tests.
00:00 +4: test/get_keys_test.dart: Keys When has master key updates a key partially                                                 
Using Meilisearch server on http://localhost:7700 for running tests.
00:00 +5: test/get_keys_test.dart: Keys When has master key updates key expiresAt                                                   
Using Meilisearch server on http://localhost:7700 for running tests.
00:00 +6: test/get_keys_test.dart: Keys When has master key deletes a key                
```

Without it, we just have a compact log that just shows up the errors and all the other information is hidden (much better).

The idea is to remove this message for every test call, but print once before the whole test suite.

I know we have the **`setUpAll`** hook, but to be feasible to use it, I think we will need to improve the whole test suite.

**Comment by brunoocasali** [comment]
To check the difference: 

<img width="655" alt="image" src="https://user-images.githubusercontent.com/4116980/155525825-b5122c2b-6143-4c7f-b8be-708993af8df8.png">


<img width="574" alt="image" src="https://user-images.githubusercontent.com/4116980/155525916-33521ee6-aab7-4231-8900-7af86927b618.png">

**Comment by michael-rodriguez22** [comment]
Just submitted a PR for this.

I passed all tests through a main `testing_suite_test.dart` file, allowing usage of the `setUpAll` hook to log the "Using Meilisearch server..." message only once, no matter which tests are run. I also added a `dart_test.yaml` configuration file to allow tests to be grouped and run by tags. 

Running the analytics tests before my changes could be done like this:

```
flutter test test/analytics_test.dart
```

After my changes, running the analytics tests can be done like this:

```
flutter test --tags "analytics"
```

I hope this solution works for y'all. If there is anything you'd like me to change, I'd be happy to keep working on this issue.

This is my first attempt at an open-source contribution, so any tips would be greatly appreciated.  Cheers!🥂
